lib/net.js

```diff
@@ -1276,12 +1276,15 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
     return;
   }
 
+  // Master listens on port and cached, if workers use different backlog,
+  // the first one take effect.
```

Comment:
I have add some documentation about the optional argument =backlog= and the behavior in cluster.markdown and net.markdown:
https://github.com/tangxinfa/node/commit/5ecefd6804fbd480de6b8f2fd1575dd8cd4beef8